File: java/benchmark_1.java
import java.util.Date;

class benchmark_1  {
    public static void main(String[] args) {
      int x=0;
      long start = new Date().getTime();
      for (int i = 0; i < 10000000; i++) {
        x = x + 1;
      }
      long end = new Date().getTime();
      long total_time = end-start;
      System.out.println("Total time=" + total_time+"ms");
    }
}

File: nodejs/benchmark_1.js
var x=0;
start = new Date().getTime();
for (var i = 0; i < 10000000; i++) {
  x = x + 1;
}
var end = new Date().getTime();
var total_time = end-start;
console.log("Total time="+total_time+"ms");